README.md

```diff
@@ -45,7 +45,9 @@ The default export of the UUID library is the
 representation _(as described in RFC-4122)_.
 
 ```js
-import uuid from 'lib:uuid'; // (Note: exact import syntax is TBD).
+// The exact way of how standard library modules will be made available (e.g. import, builtin
+// global, etc.) is out of scope for this proposal. This proposal assumes there will be *some* way
```

Comment:
To be clear; this is not out of scope for the overall proposal. I’d consider “how you get to it” a stage 2 requirement; absent builtin modules advancing, that means choosing a global name.